src/iceberg/avro/demo_avro.cc

```diff
@@ -49,18 +49,4 @@ std::string DemoAvro::print() const {
   return actual.str();
 }
```

Comment:
Get rid of demo_avro.h/cc entirely? The remaining code doesn't seem to serve any meaningful purpose.

Reply:
Good point!